### ISSUE TITLE
months go missing

### ISSUE THREAD

**Issue by shadow476z** [codrops/Calendario#53]
Hi there, First of all great plugin! thanks for sharing :).
I am using the brother or demo2 calendar of this plugin. In our app there is a reminders page with the calendar, but when ever this page loads for the second time and one navigates to the next and previous month, it skips over some of the months. For example the current month shows June, when next is clicked it hops to August and then to October...very odd and no obvious pattern.
I am thinking that the issue is with something not being reinitialized properly? Is there anyway that one can destroy the calendar control before its added to the page.?
Thanks again

**Comment by deviprsd21** [comment]
You are right but this repo is inactive and has been transferred to https://github.com/deviprsd21/Calendario, where it is being maintained and the latest version is v5.0.1,
You can get a release copy here https://github.com/deviprsd21/Calendario/releases, and there have been a lot of changes so read the read me properly!
